scene/3d/skeleton_ik_3d.cpp

```diff
@@ -330,6 +330,9 @@ void SkeletonIK3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_tip_bone", "tip_bone"), &SkeletonIK3D::set_tip_bone);
 	ClassDB::bind_method(D_METHOD("get_tip_bone"), &SkeletonIK3D::get_tip_bone);
 
+	ClassDB::bind_method(D_METHOD("set_interpolation", "interpolation"), &SkeletonIK3D::_set_interpolation);
+	ClassDB::bind_method(D_METHOD("get_interpolation"), &SkeletonIK3D::_get_interpolation);
+
```

Comment:
This and the `ADD_PROPERTY` should also be enclosed in `#ifndef DISABLE_DEPRECATED`.

I would typically move them at the end of the `_bind_method` all together in the same `#ifndef` block (both methods and property).